network/vpc/concepts.mdx

```diff
@@ -25,7 +25,7 @@ The two parts of a CIDR block are:
 
 An IPv4 CIDR block and an IPv6 CIDR block are defined for each Scaleway Private Network at the time of creation. When you attach a resource (e.g. an Instance, an Elastic Metal server or a Load Balancer) to the network, [DHCP](#dhcp) will assign one IPv4 and one IPv6 address from the designated blocks to that resource. This way, each resource automatically gets a private IPv4 and IPv6 address on the Private Network.
 
-IP addresses from within a given subnet (that is to say, allocated from a certain CIDR block which is used for a certain Private Network) display with a slash at the end. In the case of an IPv4 adddress this could look like `172.16.16.2/22`. This format encapsulates information both about the IP address itself, and which subnet it is from. If you conigure an Instance manually with this address, it sets both the correct IP and a route to the correct subnet.
+IP addresses from within a given subnet (that is to say, allocated from a certain CIDR block which is used for a certain Private Network) display with a slash at the end. In the case of an IPv4 adddress this could look like `172.16.16.2/22`. This format encapsulates information both about the IP address itself, and which subnet it is from. If you configure an Instance manually with this address, it sets both the correct IP and a route to the correct subnet.
```

Comment:
```suggestion
IP addresses from within a given subnet (that is to say, allocated from a certain CIDR block that is used for a certain Private Network) display with a slash at the end. In the case of an IPv4 address, this could look like `172.16.16.2/22`. This format encapsulates information both about the IP address itself, and which subnet it is from. If you configure an Instance manually with this address, it sets both the correct IP and a route to the correct subnet.
```